packages/manager/apps/web/client/app/hosting/translations/Messages_fr_FR.json

```diff
@@ -154,6 +154,8 @@
   "hosting_dashboard_service_privatesql_order": "Commander",
   "hosting_dashboard_service_privatesql_order_title": "Commander une base de données privée",
   "hosting_dashboard_service_privatesql_active": "Activer",
+  "hosting_dashboard_service_privatesql_active_long": "Activer une base SQL privée",
+  "hosting_dashboard_service_privatesql_active_remaining": "{{ count }} base(s) de données SQL privée(s) non activée(s)",
```

Comment:
You could have two translations, one for plural, one for singular, can't you ?

Reply:
yes but since ng-pluralize is not already used in web i don't feel to import it just for one translation, don't blame me ! ^^